proxyd/README.md

```diff
@@ -89,6 +89,59 @@ Cache use Redis and can be enabled for the following immutable methods:
 * `eth_getUncleByBlockHashAndIndex`
 * `debug_getRawReceipts` (block hash only)
 
+## Meta method `consensus_getReceipts`
+
+To support backends with different specifications in the same backend group,
+proxyd exposes a convenient method to fetch receipts abstracting away
+what specific backend will serve the request.
+
+Each backend can specify their preferred method to fetch receipts with `consensus_receipts_target`.
+
+This method takes **both** the blockNumberOrHash **and** list of transaction hashes to fetch the receipts,
+and then after selecting the backend to serve the request,
+it translates to the correct target with the appropriate parameters.
+
+Note that only one of the parameters will be actually used depending on the target.
+
+Request params
+```json
+{
+  "jsonrpc":"2.0",
+  "id": 1,
+  "params": {
+    "blockNumberOrHash": "0xc6ef2fc5426d6ad6fd9e2a26abeab0aa2411b7ab17f30a99d3cb96aed1d1055b",
+    "transactions": [
+      "0xe670ec64341771606e55d6b4ca35a1a6b75ee3d5145a99d05921026d1527331",
+      "0x88df016429689c079f3b2f6ad39fa052532c56795b733da78a91ebe6a713944b"
+    ]
+  }
```

Comment:
To bring my comment on this over from slack:
I wonder if we're better off assuming that there is some method to get receipts by block hash so we can send only the block hash, then have the op-node fall back to requesting individual receipts if there isn't actually one.
That saves us having to fetch the list of transaction hashes before we can make the call even though every provider we currently use only needs the block hash (either geth debug_getRawReceipts or alchemy alchemy_getTransactionReceipts). If none are available proxyd would just return a response indicating that.

Reply:
absolutely, I'll change the interface and submit a new revision